boot/duneboot.ml

```diff
@@ -1164,14 +1164,19 @@ module Library = struct
             let extra_flags =
               if String.starts_with ~prefix:"blake3_" fn
               then (
-                match architecture with
-                | "x86" | "i386" | "i486" | "i586" | "i686" ->
+                let is_32_bit_arch =
+                  match architecture with
+                  | "x86" | "i386" | "i486" | "i586" | "i686" -> true
```

Comment:
Isn't there something like word size in the ocaml config?

Reply:
You can't use that unfortunately. We can only use information from the ocaml config